WebApp/autoreduce_webapp/templates/instrument_variables.html

```diff
@@ -19,109 +18,136 @@ <h2>{{ instrument.name }} - Configure New Jobs</h2>
             <div class="row">
                 <div class="col-md-12 text-center">
                     <p>
-                        <a href="{% url 'runs:list' instrument=instrument.name %}">View upcoming saved configurations</a>
+                        <a href="{% url 'runs:list' instrument=instrument.name %}">
+                            View upcoming saved configurations
+                        </a>
                     </p>
                 </div>
             </div>
             <br/>
             {% if experiment_reference > 0 %}
-                <form id="instrument:variables" method="POST" action="{% url 'instrument:variables_by_experiment' instrument=instrument.name experiment_reference=experiment_reference %}" class="form-horizontal">
+                <form id="instrument_variables" method="POST"
```

Comment:
Are these form tags ok? They are neither empty tags nor do they have a closing tag.

Reply:
I decided to leave out instrument variable pages because of the current Refactor WebApp variable creation PR

Reply:
I'll note down this to look at after this and the other PR gets merged in

Reply:
Of course, I forgot about this!